video_xblock/exceptions.py

```diff
@@ -28,6 +28,14 @@ def __str__(self):
         return self.detail
 
 
+class VideoXBlockMockException(VideoXBlockException):
+    """
+    Base class for video xblock exceptions. Subclasses should provide `.default_msg` property.
+    """
+
+    default_msg = _('A Mock exception occurred.')
```

Comment:
We want it everywhere user will get this errors
In case of mock exception we do not need it but i tried to be consistent through the file